repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/services/Storage.scala

```diff
@@ -35,7 +35,7 @@ object Storage {
   private val storage = StorageOptions.getDefaultInstance.getService
 
   val TimestampFormat = DateTimeFormat.forPattern("YYYY-MM-dd-HHmmssSSS")
-  val DefaultAcl = new JArrayList(JArrays.asList(Acl.of(User.ofAllUsers, Role.READER)))
+  val DefaultAcl: JArrayList[Acl] = new JArrayList(JArrays.asList(Acl.of(User.ofAllAuthenticatedUsers, Role.READER)))
```

Comment:
Hi Anton, if we could avoid set the acls it would be even better. As the object would get the bucket default acls.
From the documentation, it seems to be possible: https://cloud.google.com/storage/docs/access-control/lists#defaultobjects

Reply:
On it! Thanks Joao!